README.md

```diff
@@ -1,7 +1,7 @@
 # Rust for Flipper Zero 🐬❤️🦀
 
 [![crates.io](https://img.shields.io/crates/v/flipperzero)](https://crates.io/crates/flipperzero)
-[![Flipper Zero API](https://img.shields.io/badge/Flipper%20Zero%20API-20.0-orange)](https://github.com/flipperdevices/flipperzero-firmware/blob/0.80.1/firmware/targets/f7/api_symbols.csv)
+[![Flipper Zero API](https://img.shields.io/badge/Flipper%20Zero%20API-26.0-orange)](https://github.com/flipperdevices/flipperzero-firmware/blob/dev/firmware/targets/f7/api_symbols.csv)
```

Comment:
Pinning the URL to `dev` means the `api_symbols.csv` won't be stable relative to the symbols used by this repo's `main` branch. This should be bumped to a firmware revision that has the API 26.0 symbols. Given that it appears the current stable firmware is only on API 23, I suggest we only bump to there for now.

Reply:
Will use 0.82.3 tag instead.